get-lm-desk.sh

```diff
@@ -468,126 +502,27 @@ function pull_models {
 
 
 #----
-# Install continue into VS Code
```

Comment:
For right now, let's not delete the VS Code + Continue stuff. I think we _should_ do that for the final script, but let's not conflate these two. I'm thinking we can/should extend `lm-desk` to be composable such that we can build different target stacks easily.

Reply:
I added it back. ✌️